axolotl-web/src/router/router.js

```diff
@@ -102,8 +102,11 @@ export const router = new Router({
 });
 
 router.beforeEach((to, from, next) => {
-
-  if (to.path === "/debug") {
+  if(to.query.token){
+    store.dispatch("setCaptchaToken",
+    to.query.token)
+  }
+  if (to.path === "/debug" || to.path ==="/verify") {
```

Comment:
Why are you adding `verify` here instead of including `captcha` below?

Reply:
Because captcha have to be generated on a specific domain https://signalcaptchas.org/registration/generate.html and the token can't be fetched as iframe. So we have to literally change the url from localhost and come back, that means we have a complete reload in between and that was needed to don't land always on the register page after comig back from the signalcaptchas.org Maybe it's not necessary anymore.

Reply:
Hm yeah I feel like the the registrationStatus is correctly set it shouldn't be needed. Can you please check?

Reply:
It works also without, it's removed now :)